core/src/test/java/org/infinispan/statetransfer/StateTransferTimestampsTest.java

```diff
@@ -0,0 +1,80 @@
+package org.infinispan.statetransfer;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.junit.Assert.assertNull;
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.assertNotNull;
+
+import org.infinispan.AdvancedCache;
+import org.infinispan.commons.time.TimeService;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.container.entries.CacheEntry;
+import org.infinispan.manager.EmbeddedCacheManager;
+import org.infinispan.test.MultipleCacheManagersTest;
+import org.infinispan.test.TestingUtil;
+import org.infinispan.util.ControlledTimeService;
+import org.testng.annotations.Test;
+
+/**
+ * Tests concurrent startup of replicated and distributed caches
+ *
+ * @author Dan Berindei
+ * @since 7.2
+ */
+@Test(testName = "statetransfer.StateTransferTimestampsTest", groups = "functional")
+public class StateTransferTimestampsTest extends MultipleCacheManagersTest {
+   public static final String CACHE_NAME = "cache";
+   private ControlledTimeService timeService;
+
+   @Override
+   public Object[] factory() {
+      return new Object[]{
+         new StateTransferTimestampsTest().cacheMode(CacheMode.REPL_SYNC),
```

Comment:
No DIST_SYNC ?

Reply:
The state transfer process is the same for all clustered caches except scattered.